src/ixdat/techniques/ec_ms.py

```diff
@@ -177,6 +183,45 @@ def ecms_calibration_curve(
             return cal, ax
         return cal
 
+    def _get_tspan_list(
+            self,
+            selector_list,
+            selector_name=None,
+            tspan_steady_pulse=0,
+    ):
+        """
+        Generate a t_span list from input of selectors.
+
+        Args:
+            selector_list (list of selector): selector numbers that define the
+                                            tspans over which data should be integrated
+            selector_name (str): selector name that should be, which by default will
+                                                refer to data['selector']
+            tspan_steady_pulse (float): length of steady state pulse period to integrate
```

Comment:
I think this should just be called `t_steady_pulse`. The use of `tspan` to me indicates it should be a `(t_start, t_end)` tuple.